txscript/script.go

```diff
@@ -37,13 +38,36 @@ const (
 	sigHashMask = 0x1f
 )
 
+const (
+	// TaprootAnnexTag is the tag for an annex. This value is used to
+	// identify the annex during tapscript spends. If there're at least two
+	// elements in the taproot witness stack, and the first byte of the
+	// last element matches this tag, then we'll extract this as a distinct
+	// item.
+	TaprootAnnexTag = 0x05
```

Comment:
Isn't this `0x50`? https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#Rationale